pkg/META

```diff
@@ -6,7 +6,7 @@ package "vfs" (
  directory = "vfs"
  description = "Datakit's VFS description"
  version = "%%VERSION%%"
- requires = ""
+ requires = "fmt astring lwt cstruct logs unix sexplib result"
```

Comment:
Maybe it's a false positive of some kind? When I comment code out, it disappears after I remove all references to `Cstruct`.